bHY6yh-d35397/cod-vyg67d_bHY6yh-d35397.xml

```diff
@@ -0,0 +1,134 @@
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-08-10" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#O" n="79-v"/>
+            <cb ed="#O" n="b"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d35397"><!-- p2l4-r14 -->
+            <head xml:id="bHY6yh-d35397-Hd1e108">Rubrica 14</head>
+            <head xml:id="bHY6yh-d35397-Hd1e111" type="question-title">De causis inducentibus pacem et concordiam</head>
+            <p xml:id="bHY6yh-d35397-d1e114">
+               De causis inducentibus 
+               <lb ed="#O"/>concordiam et pacem. Rubrica. 14
+            </p>
+            <p xml:id="bHY6yh-d35397-d1e121">
+               <lb ed="#O"/><ref xml:id="bHY6yh-d35397-Rd1e117">ARistotiles politicorum libro 5. capitulo 3</ref> <quote xml:id="bHY6yh-d35397-Qd1e129">Con%gregat 
+               etiam separatissimos co%munis 
+               timor.</quote>
+            </p>
+            <p xml:id="bHY6yh-d35397-d1e135">
+               <!--line bumps down-->
+               <lb ed="#O"/><ref xml:id="bHY6yh-d35397-Rd1e134">LUcas in eu. <!--line bumps up--> capitulo 23. et pe.</ref> <quote xml:id="bHY6yh-d35397-Qd1e139">mul<!--line bumps back down-->titudo iudeorum 
+               <lb ed="#O"/>duxerunt iesum captum ad pi%latum 
+               etcaetera. Dicens commouit po%pulum 
+               docens per uniuersam iudeam 
+               <lb ed="#O"/>et incipiens a galilea usque huc.
+               <lb ed="#O"/>Pilatus autem audiens galileam 
+               <lb ed="#O"/>interrogauit si homo galileus esset. 
+               <lb ed="#O"/>et ut cognouit quod de herodis po%testate 
+               esset remisit eum ad 
+               <lb ed="#O"/>herodem qui et ipse. Ierosolimis 
+               <lb ed="#O"/>erat illis diebus. Herodes autem 
+               <lb ed="#O"/>uisu iesu gauisus est ualde erat
+               <lb ed="#O"/>enim cupiens ex mulo tempore uidere 
+               <pb ed="#O" n="80-r"/>
+               <cb ed="#O" n="a"/>
+               <lb ed="#O"/>eum eo quod audierat multa de eo
+               <lb ed="#O"/>illo et sperabat signum aliquot ab eo 
+               <lb ed="#O"/>fieri. Interrogabat ant illum multis 
+               <lb ed="#O"/>sermonibus. At ille nichil respon%debat 
```

Comment:
i think you've got an error in your line break placement; please double check and confirm my suggestion
```suggestion
               <lb ed="#O"/>sermonibus. At ille nichil respon<lb ed="#O" break="no"/>debat 
```

Reply:
I double checked and "respondebat" is indeed one word, there is a faint dash line after "respon"